src/GitVersion.Core/Git/CommitExtensions.cs

```diff
@@ -0,0 +1,8 @@
+using GitVersion.Extensions;
+
+namespace GitVersion;
+
+public static class CommitExtensions
+{
+    public static bool IsMergeCommit(this ICommit source) => source.NotNull().Parents.Count >= 2;
```

Comment:
It was a fix for a unit test where the ICommit was mocked. The unit test failed because the expression `Parents.Count >= 2` was not executed but the default value (false) returned. This logic is actually not a implementation detail of ICommit and that's why I have moved it to a extensions class.

Reply:
I'm not sure if we need the ICommit interface at all. You can think of the a Commit as data transfer object without any implementation detail.